planning/behavior_path_planner/include/behavior_path_planner/scene_module/lane_change/base_class.hpp

```diff
@@ -110,6 +114,54 @@ class LaneChangeBase
     return isAbortState() ? *abort_path_ : status_.lane_change_path;
   }
 
+  PathWithLaneId getExtendPath(const PathWithLaneId & path) const
```

Comment:
```suggestion
 virtual PathWithLaneId getExtendPath(const PathWithLaneId & path) const = 0
```

I would suggest to move this to by overriding the function in `normal.hpp/normal.cpp`.
The reason is that the code implementation is too long for `base_class.hpp` and I don't want to flood this file with long implementations.